src/main/java/core/basesyntax/FigureDrawer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawer {
```

Comment:
Using **Figure** in the interface name is OOP Abstraction principle violation. What do we split class and interface for if we use  **Figure** in the interface name? The idea is that any other class can implement this interface as well.
```suggestion
public interface Drawable {
```